### PR TITLE
Improve build instructions

### PR DESCRIPTION
**Pull request by natke**

### CODE REVIEW

File: docs/genai/howto/build-from-source.md [hunk 1]
```diff
@@ -87,9 +87,10 @@ Build from source and copy the include and libraries into `ORT_HOME`
 On Windows
 
 ```cmd
-build.bat --build_shared_lib --skip_tests --parallel [--use_cuda]
+build.bat --build_shared_lib --skip_tests --parallel [--use_cuda] --config Release
 copy include\onnxruntime\core\session\onnxruntime_c_api.h <ORT_HOME>\include
 copy build\Windows\Debug\Debug\*.dll <ORT_HOME>\lib
+copy build\Windows\Debug\Debug\onnxruntime.lib <ORTHOME>\lib
```
Comment (yufenglee) [inline docs/genai/howto/build-from-source.md:93]: build\Windows\Release\Release

File: docs/genai/howto/build-from-source.md [hunk 2]
```diff
@@ -87,9 +87,10 @@ Build from source and copy the include and libraries into `ORT_HOME`
 On Windows
 
 ```cmd
-build.bat --build_shared_lib --skip_tests --parallel [--use_cuda]
+build.bat --build_shared_lib --skip_tests --parallel [--use_cuda] --config Release
 copy include\onnxruntime\core\session\onnxruntime_c_api.h <ORT_HOME>\include
 copy build\Windows\Debug\Debug\*.dll <ORT_HOME>\lib
```
Comment (yufenglee) [inline docs/genai/howto/build-from-source.md:92]: same here

Comment (yufenglee) [reply docs/genai/howto/build-from-source.md:92]: maybe change Release to RelWithDebInfo to keep consistent with Linux

Comment (natke) [reply docs/genai/howto/build-from-source.md]: So we can do RelWithDebInfo on Windows too?

**Review by yufenglee — APPROVED** [review]

**Pull request merged**